playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -27,16 +27,8 @@
         embedded_etcd: "{{ groups.oo_etcd_to_config | default([]) | length == 0 }}"
         debug_level: "{{ openshift_master_debug_level | default(openshift.common.debug_level | default(2)) }}"
 
-- name: Upgrade and backup etcd
-  include: ./etcd/main.yml
-
-- name: Upgrade master packages
-  hosts: oo_masters_to_config
-  roles:
-  - openshift_facts
-  tasks:
-  - include: rpm_upgrade.yml component=master
-    when: not openshift.common.is_containerized | bool
+  #- name: Upgrade and backup etcd
```

Comment:
In the future we should consider splitting etcd upgrade out of control plane, it's really more like "Database". But fine to keep it for now until we find that it creates problems.